### PR TITLE
Matt additions to Ansible lesson

### PR DESCRIPTION
**Pull request by Mierdin**
# TODO

- Figure out takeaways for last stage
- OSPF doesn't seem to work for me in last stage

### CODE REVIEW

**Comment by IPvSean** [comment]
I like the changes @Mierdin , thanks for cleaning that up behind me

File: lessons/tools/lesson-41-ansible-network/stage4/guide.md [hunk 1]
```diff
@@ -178,11 +176,7 @@ You have completed stage 4!
 
 ## Takeaways
 
-- Resource modules and facts have a direct relationship allowing Ansible to read existing brownfield networks and create a source of truth really quickly.
-- The `ansible_network_resources` parameter is used to collect facts around a specific resource such as l3_interfaces.
-- Variables are mostly commonly stored in group_vars and host_vars. This short example only used host_vars.
-
-For more information on why resource modules were developed please refer to [this blog post](https://www.ansible.com/blog/network-features-coming-soon-in-ansible-engine-2.9).
+< TAKEAWAYS >
```
Comment (IPvSean) [inline lessons/tools/lesson-41-ansible-network/stage4/guide.md]: - [Jinja2](https://docs.ansible.com/ansible/latest/user_guide/playbooks_templating.html) templates allow us to easily template out a device configuration.
- The *os_config (e.g. [junos_config](https://docs.ansible.com/ansible/latest/modules/junos_config_module.html)) and cli_config modules can source a jinja2 template file, and push directly to a device. If you want to just render a configuration locally on the control node, or template to a Linux device, use the [template module](https://docs.ansible.com/ansible/latest/modules/template_module.html).
- Variables are mostly commonly stored in `group_vars` and `host_vars`. This short example only used `host_vars`.

**Comment by IPvSean** [comment]
> OSPF doesn't seem to work for me in last stage

we just need to turn on the ospfd daemon on cumulus, I thought I did a PR on that for some reason.

```
antidote@cumulus:mgmt-vrf:~$ cat /etc/frr/daemons
zebra=yes
bgpd=yes
```

should be like

```
antidote@cumulus:mgmt-vrf:~$ cat /etc/frr/daemons
zebra=yes
bgpd=yes
ospfd=yes ospfd_instances="1"
```

I thought I had already added that, sorry about that

I figured the lesson was long enough without doing the above in ansible, I didn't want to teach people about frr daemon processes if it was not required

**Comment by Mierdin** [comment]
No worries, we have a lesson on FRR in the pipeline, just not published. So we don't need to do it in the foreground if we don't want to, I'll add that to the inter-stage configuration for `cvx1`

**Pull request merged**